ledis/Controllers/SignUpViewController.swift

```diff
@@ -24,90 +24,45 @@ class SignUpViewController: UIViewController {
-        emailTextField.layer.cornerRadius = 21.0
-        emailTextField.clipsToBounds = true
-        emailTextField.layer.borderWidth = 0.8
-        emailTextField.layer.borderColor = UIColor.black.cgColor
-        passwordTextField.layer.cornerRadius = 21.0
-        passwordTextField.clipsToBounds = true
-        passwordTextField.layer.borderWidth = 0.8
-        passwordTextField.layer.borderColor = UIColor.black.cgColor
-        
         usernameTextField.addTarget(self, action: #selector(handleTextInputChange), for: .editingChanged)
         emailTextField.addTarget(self, action: #selector(handleTextInputChange), for: .editingChanged)
         passwordTextField.addTarget(self, action: #selector(handleTextInputChange), for: .editingChanged)
-        
-        //padding View for username text field
-        let paddingViewUsername = UIView(frame: CGRect(x: 0, y: 0, width: 15, height: self.usernameTextField.frame.height))
-        usernameTextField.leftView = paddingViewUsername
-        usernameTextField.leftViewMode = UITextFieldViewMode.always
-        
-        //padding View for email text field
-        let paddingViewEmail = UIView(frame: CGRect(x: 0, y: 0, width: 15, height: self.emailTextField.frame.height))
-        emailTextField.leftView = paddingViewEmail
-        emailTextField.leftViewMode = UITextFieldViewMode.always
-        
-        //padding View for login password field
-        let paddingViewPassword = UIView(frame: CGRect(x: 0, y: 0, width: 15, height: self.usernameTextField.frame.height))
-        passwordTextField.leftView = paddingViewPassword
-        passwordTextField.leftViewMode = UITextFieldViewMode.always
-        
     }
     
     @objc func handleTextInputChange () {
         let isFormValid = usernameTextField.text?.count ?? 0 > 0 && emailTextField.text?.count ?? 0 > 0 && passwordTextField.text?.count ?? 0 > 0
-        if isFormValid && isBoxClicked { // have a question, how to do it correctly
-            signUpButton.isEnabled = true
-            signUpButton.backgroundColor = UIColor.rgb(red: 48, green: 79, blue: 254)
-        }
-        else {
-            signUpButton.isEnabled = false
-            signUpButton.backgroundColor = UIColor.rgb(red: 112, green: 131, blue: 241)
+        if isBoxClicked{
+            signUpButton.isEnabled = isFormValid
         }
+        signUpButton.backgroundColor = isFormValid ? StyleManager.Colors.validInput : StyleManager.Colors.invalidInput
     }
     
     @IBAction func handleSignUp(_ sender: UIButton) {
-        guard let username = usernameTextField.text, username.count > 0 else { return }
-        guard let email = emailTextField.text, email.count > 0 else { return }
-        guard let password = passwordTextField.text, password.count > 0  else { return }
+        guard let username = usernameTextField.text, username.count > 0, let email = emailTextField.text, email.count > 0, let password = passwordTextField.text, password.count > 0 else { return }
```

Comment:
if you have such long guard it will be much better to write it like this: 
```swift
guard let username = usernameTextField.text, username.count > 0,
          let email = emailTextField.text, email.count > 0,
          let password = passwordTextField.text, password.count > 0
else {
    return 
}          
```
such code structure is more readable